src/modules/normalize/filter_volume.c

```diff
@@ -413,15 +419,15 @@ static mlt_frame filter_process( mlt_filter filter, mlt_frame frame )
 		{
 			amplitude *= mlt_filter_get_progress( filter, frame );
 		}
-		mlt_properties_set_int( instance_props, "normalise", 1 );
+		mlt_properties_set_int( instance_props, "normalize", 1 );
```

Comment:
I think we need to set both of these properties, or I would prefer to revert the property name change. It is not important enough to change it IMO.

Reply:
Done